auth/custom_claims.js

```diff
@@ -7,58 +7,72 @@ let idToken;
+    // next time a new one is issued.
+    });
 // [END set_custom_user_claims]
 
-
 // [START verify_custom_claims]
 // Verify the ID token first.
-admin.auth().verifyIdToken(idToken).then((claims) => {
-  if (claims.admin === true) {
-    // Allow access to requested admin resource.
-  }
-});
+admin
+    .auth()
+    .verifyIdToken(idToken)
+    .then((claims) => {
+      if (claims.admin === true) {
+      // Allow access to requested admin resource.
+      }
+    });
 // [END verify_custom_claims]
 
 // [START read_custom_user_claims]
 // Lookup the user associated with the specified uid.
-admin.auth().getUser(uid).then((userRecord) => {
-  // The claims can be accessed on the user record.
-  console.log(userRecord.customClaims.admin);
-});
+admin
+    .auth()
+    .getUser(uid)
+    .then((userRecord) => {
+    // The claims can be accessed on the user record.
+      console.log(userRecord.customClaims.admin);
+    });
 // [END read_custom_user_claims]
 
 // [START set_custom_user_claims_script]
-admin.auth().getUserByEmail('user@admin.example.com').then((user) => {
-  // Confirm user is verified.
-  if (user.emailVerified) {
-    // Add custom claims for additional privileges.
-    // This will be picked up by the user on token refresh or next sign in on new device.
-    return admin.auth().setCustomUserClaims(user.uid, {
-      admin: true
+admin
+    .auth()
+    .getUserByEmail('user@admin.example.com')
+    .then((user) => {
+    // Confirm user is verified.
+      if (user.emailVerified) {
+      // Add custom claims for additional privileges.
+      // This will be picked up by the user on token refresh or next sign in on new device.
+        return admin.auth().setCustomUserClaims(user.uid, {
+          admin: true,
+        });
+      }
+    })
+    .catch((error) => {
+      console.log(error);
     });
-  }
-})
-  .catch((error) => {
-    console.log(error);
-  });
 // [END set_custom_user_claims_script]
 
 // [START set_custom_user_claims_incremental]
-admin.auth().getUserByEmail('user@admin.example.com').then((user) => {
-  // Add incremental custom claim without overwriting existing claims.
-  const currentCustomClaims = user.customClaims;
-  if (currentCustomClaims.admin) {
-    // Add level.
-    currentCustomClaims['accessLevel'] = 10;
-    // Add custom claims for additional privileges.
-    return admin.auth().setCustomUserClaims(user.uid, currentCustomClaims);
-  }
-})
-  .catch((error) => {
-    console.log(error);
-  });
-// [END set_custom_user_claims_incremental]
\ No newline at end of file
+admin
+    .auth()
+    .getUserByEmail('user@admin.example.com')
+    .then((user) => {
+    // Add incremental custom claim without overwriting existing claims.
+      const currentCustomClaims = user.customClaims;
+      if (currentCustomClaims.admin) {
+      // Add level.
+        currentCustomClaims['accessLevel'] = 10;
+        // Add custom claims for additional privileges.
```

Comment:
This is what I expected to see above. There are ultiple similar issues across the code, I won't comment on them all to reduce comments on this PR.